RallyHereIntegration/Source/RallyHereGameHostProvider/Public/RH_GameHostAdapter.h

```diff
@@ -9,9 +9,14 @@
 // this is done here rather than at point of use, as the UHT / UBT system wants to treat this as a "first party" file and enforce unreal include orders
 THIRD_PARTY_INCLUDES_START
 
+#if PLATFORM_WINDOWS
 // define RH_EXPORT macro into include mode
 #define RH_EXPORT __declspec(dllimport)
 #define RH_CPP_EXPORT __declspec(dllimport)
+#else
+#define RH_EXPORT
+#define RH_CPP_EXPORT
```

Comment:
I believe the proper way to do this for non-windows is by setting `__attribute__((visibility("default")))` for both import and export.  I'll see about making a PR on game-host-adapter to make this the default behavior

Reply:
My understanding is that attribute visibility tags are meaningless on import of shared objects, so I've left imports being blank when dealing with them.